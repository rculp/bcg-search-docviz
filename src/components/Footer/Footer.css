.Footer {
  font-style: italic;
  width: 100%;
  padding-bottom: 1em;
}