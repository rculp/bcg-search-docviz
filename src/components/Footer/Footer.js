import React from 'react';

import './Footer.scss';

const Footer = () => (
  <div id="footer">
    Here is a sticky footer. If there is not enough content above, it will remain at the
    bottom of the page.
    <br />
    <br />
    If there is enough content above, it will move below the bottom of the viewport. Note
    that this behavior remains consistent even when the footer is given padding. It also
    obeys the bottom padding of the container above if it wishes to add extra space before
    encountering the footer.
  </div>
);

export default Footer;
