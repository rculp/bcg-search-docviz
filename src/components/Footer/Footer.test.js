import { shallow } from 'enzyme';

import React from 'react';

import Footer from './Footer';

describe(Footer, () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Footer />
    );
    expect(component).toMatchSnapshot();
  });
});