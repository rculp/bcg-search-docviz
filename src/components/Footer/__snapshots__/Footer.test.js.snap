// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`() =>
_react2.default.createElement('footer', { __source: { fileName: _jsxFileName, lineNumber: 6 }, __self: undefined }, 'Footer') renders and matches our snapshot 1`] = `
<footer>
  Footer
</footer>
`;
